### ISSUE TITLE
How can I detect the installed sqlean version?

### ISSUE THREAD

**Issue by iainelder** [nalgeon/sqlean#47]
When downloading from GitHub releases I can get the version number from the releases API in the `name` and `tag_name` fields.

Unless I explicitly save that info in a text file next to the .so files, I don't see how to detect the version of of sqlean after installation.

Is there another way to decode those files that I've missed?

**Comment by nalgeon** [comment]
I don't think so. Why do you need it?

**Comment by iainelder** [comment]
It's just to know which version of sqlean I have installed when I come back later to use it for analysis.

Not knowing the installed version isn't a big problem, though, because generally I would like to stay on the latest version. I have a [script](https://github.com/iainelder/dotfiles/blob/master/programs/sqlean/install.bash) to automate the installation, so when I need to use sqlean I can just run my script to update it to the latest version.

It's fine for me if you don't want to do anything about this. I can solve it myself as I described.
